src/rbd.cc

```diff
@@ -2385,7 +2741,8 @@ if (!set_conf_param(v, p1, p2, p3)) { \
       opt_cmd != OPT_IMPORT &&
       opt_cmd != OPT_IMPORT_DIFF &&
       opt_cmd != OPT_UNMAP &&
-      opt_cmd != OPT_SHOWMAPPED && !imgname) {
+      opt_cmd != OPT_SHOWMAPPED &&
+      opt_cmd != OPT_MERGE_DIFF && !imgname) {
```

Comment:
> it doesn't look like image name is required

Yes, so when the opt_cmd is OPT_MERGE_DIFF, this code will ignore to check imgname.